app/src/components/tables/PeersTable/PeersTable.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  getPeersTableOptions,
+  getTotalPeers,
+  setPeerTableOptions,
+  updatePeerPageNum,
+  useAppSelector,
+} from 'src/store';
+import styled from '@emotion/styled';
+import { ColumnDef } from '@tanstack/react-table';
+import { ApiData } from 'src/api/types';
+import { colors, fontWeight } from 'src/styled-theme';
+import { SelectOptions } from 'src/components/layout/Header/Partials';
+import { Table } from '../../base';
+import { NumberedPagination } from '../Pagination/NumberedPagination';
+
+interface PeersTableProps {
+  readonly peers: ApiData.Peer[];
+  isTableLoading: boolean;
+  setIsTableLoading: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export const PeersTable: React.FC<PeersTableProps> = ({
+  peers,
+  isTableLoading,
+  setIsTableLoading,
```

Comment:
Could this just be internal state within the component?  See the [ValidatorsTable](https://github.com/casper-network/casper-blockexplorer-frontend/pull/214/files#diff-57a13a532071684f7d61d97b5a43f58f81dd8071ffb5845ca81367c76a848b41)

Would be good to keep this pattern consistent if possible